src/modules/facility-loan/facility-loan.service.ts

```diff
@@ -55,6 +84,152 @@ export class FacilityLoanService {
     return this.acbsAuthenticationService.getIdToken();
   }
 
+  private getBaseMessage(facilityIdentifier: UkefId, newFacilityLoan: FacilityLoanToCreate) {
+    let loanInstrumentCode;
+    if (newFacilityLoan.productTypeGroup === ENUMS.PRODUCT_TYPE_GROUPS.GEF) {
+      loanInstrumentCode = ENUMS.PRODUCT_TYPE_IDS.GEF_CASH;
+    } else if (newFacilityLoan.productTypeGroup === ENUMS.PRODUCT_TYPE_GROUPS.BOND) {
+      loanInstrumentCode = ENUMS.PRODUCT_TYPE_IDS.BSS;
+    } else {
+      loanInstrumentCode = ENUMS.PRODUCT_TYPE_IDS.EWCS;
+    }
```

Comment:
Yes, I was a bit unsure about this because not all of the ProductTypeIds were covered in the MuleSoft logic, eg. `ProductTypeId` can have value `281` but `loanInstrumentCode` would never have that. I'll raise on Slack.

Reply:
Because loanInstrumentCode  and productTypeId are very similar, maybe similar solution can be used to get loanInstrumentCode.

`let loanInstrumentCode = (productTypeId === '281') ? '280': productTypeId;`